src/EA.Weee.DataAccess/DataAccess/EvidenceDataAccess.cs

```diff
@@ -289,7 +289,7 @@ public async Task<List<NoteTonnage>> GetTonnageByIds(List<Guid> ids)
             }
 
             var itemsToRemove =
-                note.NoteTransferTonnage.Where(ntt => tonnages.All(t => t.NoteTonnageId != ntt.NoteTonnageId));
+                note.NoteTransferTonnage.Where(ntt => tonnages.All(t => t.NoteTonnageId != ntt.NoteTonnageId) || (((ntt.Received == 0.00m && ntt.Reused == 0.00m) || (ntt.Received == null && ntt.Reused == null)) && status.Equals(NoteStatus.Submitted)));
```

Comment:
@NateHogan Wouldnt bother checking the reused values. If the Recieved is null or zero just remove as reused can't have value if received doesnt

Reply:
Updated